java/flight/flight-core/src/main/java/org/apache/arrow/flight/SetSessionOptionsResult.java

```diff
@@ -45,12 +45,14 @@ static ErrorValue fromProtocol(Flight.SetSessionOptionsResult.ErrorValue s) {
       return values()[s.getNumber()];
     }
 
+    @SuppressWarnings("EnumOrdinal")
```

Comment:
IMO, the right way to do this would be to add a constructor and store the Protobuf enum value in the enum instance

Reply:
Right, that is the ideal way. Let me track down and change the similar other instances.